requirements/dev.txt

```diff
@@ -1,9 +1,9 @@
-autopep8 ~= 2.0.1
-coverage ~= 7.1.0
-flake8 ~= 6.0
-ipython ~= 7.32
-isort ~= 5.5
-moto ~= 1.3
-pytest ~= 7.2.1
-python-dotenv ~=0.21
-safety ~= 1.10
+autopep8 ~= 2.0.4
+coverage ~= 7.3.2
+flake8 ~= 6.1.0
+ipython ~= 8.17.2
```

Comment:
There was also a medium severity vulnerability in `ipython`. That should now be squashed.

There was one last medium severity vulnerability in `setuptools`, but when I ran things locally, the new versions of everything didn't look like they included `setuptools`, so that should also be squashed.